src/translate/translate.py

```diff
@@ -332,7 +332,8 @@ def build_sas_operator(name, condition, effects_by_variable, cost, ranges,
             # precondition, so we remove it from the prevail condition
             condition.pop(var, -1)
     if not pre_post:  # operator is noop
-        return None
+        if options.filter_noop_operators:
+            return None
```

Comment:
Combine into a single `if` as in the other places: `if not pre_post and not options.keep_no_ops:`